test/integration/install_scripts_int_test.go

```diff
@@ -157,14 +157,14 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstall_NonEmptyTarget() {
 	script := scriptPath(suite.T(), ts.Dirs.Work)
 	argsPlain := []string{script, "-t", ts.Dirs.Work}
 	argsPlain = append(argsPlain, "-b", constants.BranchName)
-	var cp *termtest.ConsoleProcess
+	var cp *e2e.SpawnedCmd
 	if runtime.GOOS != "windows" {
-		cp = ts.SpawnCmdWithOpts("bash", e2e.WithArgs(argsPlain...))
+		cp = ts.SpawnCmdWithOpts("bash", e2e.OptArgs(argsPlain...))
 	} else {
-		cp = ts.SpawnCmdWithOpts("powershell.exe", e2e.WithArgs(argsPlain...), e2e.AppendEnv("SHELL="))
+		cp = ts.SpawnCmdWithOpts("powershell.exe", e2e.OptArgs(argsPlain...), e2e.OptAppendEnv("SHELL="))
 	}
-	cp.ExpectLongString("Installation path must be an empty directory")
-	cp.Wait()
+	cp.Expect("Installation path must be an empty directory")
+	cp.ExpectExit()
```

Comment:
I should've added a comment, cause I don't remember the specifics. But looking at the commit and the context I believe I did this because the exit code doesn't bubble up properly through powershell.

Reply:
I think it's better to have some comment here rather than none at all. If I saw this down the road I would be very tempted to change it to `ExpectExitCode(0)`.